mlem/cli/config.py

```diff
@@ -32,9 +32,8 @@ def config_set(
     ),
 ):
     """Set configuration value
-
-    Examples:
-        $ mlem config set pandas.default_format csv
+    \n\n
```

Comment:
??

Reply:
And if you replace those with actual newline characters?

Reply:
Funny, it works. Stupid me)